### PR TITLE
Wrote function to purge accounts which aren't associated with the max…

### PR DESCRIPTION
**Pull request by ba205**
…imum block level, made iterations of purging configurable
#61

### CODE REVIEW

File: src/main/scala/tech/cryptonomic/conseil/Lorre.scala [hunk 1]
```diff
@@ -33,9 +35,13 @@ object Lorre extends App with LazyLogging {
       if (shouldFeeBeCalculatedThisIteration) {
         FeeOperations.processTezosAverageFees()
       }
+      if (shouldAccountsBePurgedThisIteration) {
```
Comment (vishakh) [inline src/main/scala/tech/cryptonomic/conseil/Lorre.scala]: Can we not just change the condition to ```iterationsOfLorre % purgeAccountsInterval == 0```?

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala [hunk 2]
```diff
@@ -199,4 +200,15 @@ object TezosDatabaseOperations {
     }
   }
 
+  def purgeOldAccounts(): Try[Int] = {
```
Comment (vishakh) [inline src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala]: Please document this function.

File: src/main/scala/tech/cryptonomic/conseil/Lorre.scala (hunk 1 above)
Comment (ba205) [reply src/main/scala/tech/cryptonomic/conseil/Lorre.scala]: Should "shouldFeeBeCalculatedThisIteration" follow the same treatment, for consistency?

Comment (vishakh) [reply src/main/scala/tech/cryptonomic/conseil/Lorre.scala]: Yes, please.

Comment (ba205) [reply src/main/scala/tech/cryptonomic/conseil/Lorre.scala]: On it.

**Pull request merged**